wowup-electron/src/app/pages/my-addons/my-addons.component.html

```diff
@@ -126,6 +126,12 @@ <h1>{{ "COMMON.SEARCH.NO_ADDONS" | translate }}</h1>
         {{ "PAGES.MY_ADDONS.ADDON_CONTEXT_MENU.AUTO_UPDATE_ADDON_BUTTON" | translate }}
       </mat-checkbox>
     </div>
+    <!-- AUTO UPDATE NOTIFICATIONS -->
+    <div *ngIf="canSetAutoUpdateNotifications(listItem)" class="mat-menu-item"> 
+      <mat-checkbox [checked]="listItem.addon.autoUpdateNotificationsEnabled" (click)="onClickAutoUpdateAddonNotifications(listItem)">
+        {{ "PAGES.MY_ADDONS.ADDON_CONTEXT_MENU.IGNORE_AUTO_UPDATE_ADDON_NOTIFICATIONS_BUTTON" | translate }}
```

Comment:
I think this text needs to be changed from Ignore Notifications to Notifications Enabled. Since the checkbox is the opposite of the text.

Reply:
Key has now been changed to AUTO_UPDATE_ADDON_NOTIFICATIONS_ENABLED_BUTTON and with the suggested text as value, good catch